apps/flame-defi/app/swap/use-token-modal.tsx

```diff
@@ -0,0 +1,43 @@
+import { CelestiaIcon, MilkTiaIcon, StrideTiaIcon, UsdcIcon, WrappedTiaIcon } from "@repo/ui/icons";
+import { IconProps } from "@repo/ui/types";
+
+export interface TokenItem {
+    Icon: React.ComponentType<IconProps>;
+    title: string;
+    symbol: string;
+  }
+  
+  // NOTE: temporary tokens until we have a real token list from a api
+  const tokens: TokenItem[] = [
+    {
+      Icon: CelestiaIcon,
+      title: "TIA",
+      symbol: "TIA",
+    },
+    {
+      Icon: WrappedTiaIcon,
+      title: "Wrapped Celestia",
+      symbol: "WTIA",
+    },
+    {
+      Icon: MilkTiaIcon,
+      title: "Milk TIA",
+      symbol: "milkTIA",
+    },
+    {
+      Icon: StrideTiaIcon,
+      title: "Stride TIA",
+      symbol: "stTIA",
+    },
+    {
+      Icon: UsdcIcon,
+      title: "USDC",
+      symbol: "USDC",
+    },
+  ];
+
+  export function useTokenModal() {
```

Comment:
Let's rename this file to `useTokenModal` for now to be consistent with our other file naming. We can definitely chat about preferred naming, but for now let's just be consistent.